draft-irtf-t2trg-rest-iot.md

```diff
@@ -283,16 +281,18 @@ Uniform Resource Identifier (URI):
 : A global identifier for resources. 
 See {{sec-uris}} for more details.
 
+
 # Basics
 
 ## Architecture {#sec-architecture}
 
 The components of a RESTful system are assigned one or both of two roles: client or server.
-Note that the terms "client" and "server" refer only to the roles that the nodes assume for a particular message exchange. The same node might act as a client in some communications and a server in others.
+Note that the terms "client" and "server" refer only to the roles that the nodes assume for a particular message exchange.
+The same node might act as a client in some communications and a server in others.
 Classic user agents (e.g., Web browsers) are always in the client role and have the initiative to issue requests.
 Origin servers always have the server role and govern over the resources they host.
 Simple IoT devices, such as sensors and actuators, are commonly acting as servers and exposing their physical world interaction capabilities (e.g., temperature measurement or door lock control capability) as resources.
-Typical IoT system client can be a cloud service that retrieves data from the sensors and commands the actuators based on the sensor information.
+Typical IoT system clients can be a cloud service that retrieves data from the sensors and commands the actuators based on the sensor information.
```

Comment:
Should also "cloud services" be plural here?

Reply:
I took the too easy fix... needed a change to "A typical ..." -- done now.